docs/config.md

```diff
@@ -1467,23 +1467,34 @@ On all platforms, the user's global `jj` configurations are loaded in the follow
 precedence order (with later configs overriding earlier ones):
```

Comment:
also might want to mention "unless `JJ_CONFIG` is set" somewhere around here.

Reply:
That's already mentioned on line 1499

Reply:
(I did this edit)

Yes, but not everyone will read all the way down to there. Also, I'd say that strictly speaking, this statement is simply incorrect without a caveat.